### PR TITLE
Add initial support for sqlcommenter

### PR DESCRIPTION
**Pull request by telser**

### CODE REVIEW

File: orville-postgresql-libpq/src/Orville/PostgreSQL/Internal/SqlCommenter.hs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Module    : Orville.PostgreSQL.Internal.SqlCommenter
+Copyright : Flipstone Technology Partners 2016-2022
+License   : MIT
+
+Stability : unstable
+
+This module provides the very basics for [sqlcommenter](https://google.github.io/sqlcommenter)
+support.
+-}
+module Orville.PostgreSQL.Internal.SqlCommenter (SqlCommenter, addComment) where
```
Comment (qxjit) [inline orville-postgresql-libpq/src/Orville/PostgreSQL/Internal/SqlCommenter.hs:12]: Can we change this to multi-line exports?

File: orville-postgresql-libpq/test/Test/SqlCommenter.hs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  Property.singletonNamedProperty "SqlCommenter is escaped and put at end of raw sql" $ do
+    let rawSql :: RawSql.RawSql
+        rawSql =
+          SqlCommenter.addComment staticSqlCommenter $
+            RawSql.fromString "SELECT * "
+              <> RawSql.fromString "FROM foo "
+              <> RawSql.fromString "WHERE id = 1"
+
+        expectedBytes =
+          B8.pack "SELECT * FROM foo WHERE id = 1/*key='value',keyForEscapedValue='queryParam%3Dfoo%20bar%2Fbaz-fizz',keyWith%27InIt='valueWith%27InIt',orm='orville'*/"
```
Comment (qxjit) [inline orville-postgresql-libpq/test/Test/SqlCommenter.hs:46]: Maybe we want a space before the comment block inside the SQL?

File: orville-postgresql-libpq/test/Test/SqlCommenter.hs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+import qualified Orville.PostgreSQL.Internal.RawSql as RawSql
+import qualified Orville.PostgreSQL.Internal.SqlCommenter as SqlCommenter
+
+import Test.Expr.TestSchema
+  ( assertEqualFooBarRows, -- barColumn,
+    dropAndRecreateTestTable,
+    findAllFooBars,
+    fooBarTable, -- fooColumn,
+    insertFooBarSource,
+    mkFooBar,
```
Comment (qxjit) [inline orville-postgresql-libpq/test/Test/SqlCommenter.hs:23]: Stray comments in this import block?

File: orville-postgresql-libpq/src/Orville/PostgreSQL/Internal/SqlCommenter.hs (hunk 1 above)
Comment (telser) [reply orville-postgresql-libpq/src/Orville/PostgreSQL/Internal/SqlCommenter.hs:12]: I tried to force it and it was put back :( I'll try again

File: orville-postgresql-libpq/test/Test/SqlCommenter.hs (hunk 3 above)
Comment (telser) [reply orville-postgresql-libpq/test/Test/SqlCommenter.hs:26]: Should be updated now.

File: orville-postgresql-libpq/test/Test/SqlCommenter.hs (hunk 2 above)
Comment (telser) [reply orville-postgresql-libpq/test/Test/SqlCommenter.hs]: Done!

**Review by qxjit — APPROVED** [review]
Looks good, aside from the formatting build failing on the most recent commit

**Pull request merged**